docs/uitour.rst

```diff
@@ -48,6 +48,9 @@ Note that ``browser.uitour.testingOrigins`` can be a comma separated list of dom
     must also exclude the domain protocol e.g. ``https://``. A browser restart is also required
     after adding an allowed domain.
 
+    If you are working on Firefox Accounts integration, you can use the ``identity.fxaccounts.autoconfig.uri``
+    config property.
```

Comment:
Trying to read this doc with my beginner's hat on: what exactly would I use this property to *do*, and *why*? Ideally I think this would say this would say something like "use the blah-bah config property to use a locally-running FxA server" and link to FxA docs with more details about doing so.